bin/postinst-qa-check.d/50gnome2-utils

```diff
@@ -27,15 +27,16 @@ gnome2_icon_cache_check() {
 		# (note: yes, it will eagerly repeat the update for next dirs
```

Comment:
This comment is no longer valid now.

Reply:
Also fixed in 5fc7281657e2a5a4b22e969853364b142a37d05f.